releasenotes/notes/0.25/add-block-collection-options-359d5e496313acdb.yaml

```diff
@@ -1,18 +1,20 @@
 ---
-features:
+features_circuits:
```

Comment:
I'm wondering if this should be the transpiler because it's only for the dagcircuit and only really used for the collection passes.

Reply:
I went through and re-categorized all of the `dagcircuit`-related changes as "transpiler". I'll push up the commit shortly with other review changes.